Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -380,11 +453,136 @@ private class GeometryEditorModel: ObservableObject {
     ///   - tool: The tool to draw with.
     ///   - geometryType: The type of geometry to draw.
     func startEditing(with tool: GeometryEditorTool, geometryType: Geometry.Type) {
+        selectedGraphic = nil
         configureGeometryEditorTool(tool, scaleMode: scaleMode)
         geometryEditor.tool = tool
         geometryEditor.start(withType: geometryType)
         isStarted = true
     }
+    
+    /// Starts editing a given graphic with the geometry editor.
+    /// - Parameter graphic: The graphic to edit.
+    func startEditing(withGraphic graphic: Graphic) {
+        guard let geometry = graphic.geometry else { return }
```

Comment:
```suggestion
        let geometry = graphic.geometry!
```
Since we can tap on the graphic, the geometry must be present.